Owncloud iOs Client/InstantUpload/InstantUpload.m

```diff
@@ -0,0 +1,215 @@
+//
+//  InstantUpload.m
+//  Owncloud iOs Client
+//
+//  Created by Jon Schneider on 5/20/16.
+//
+
+#import <CoreLocation/CoreLocation.h>
+#import <Photos/Photos.h>
+
+#import "InstantUpload.h"
+
+#import "AppDelegate.h"
+#import "constants.h"
+
+#import "ManageAppSettingsDB.h"
+#import "ManageUsersDB.h"
+#import "UserDto.h"
+
+#define kPhotoLibraryAccessPermissionDeniedAlert [[UIAlertView alloc] initWithTitle:NSLocalizedString(@"access_photos_library_not_enabled", nil) message:NSLocalizedString(@"message_access_photos_not_enabled", nil) delegate:nil cancelButtonTitle:NSLocalizedString(@"ok", nil) otherButtonTitles:nil]
+
+//TODO: Language should be for background upload, not general Instant Upload
+#define kLocationAccessAlwaysPermissionDeniedAlert [[UIAlertView alloc] initWithTitle:NSLocalizedString(@"location_not_enabled", nil) message:NSLocalizedString(@"message_location_not_enabled", nil) delegate:nil cancelButtonTitle:NSLocalizedString(@"ok", nil) otherButtonTitles:nil]
```

Comment:
About the languages as @nasli said before. It is just necessary to add the English one. The other will be send to be translated on Transifex once this Pull Request arrive to the "master" branch.